test/Java8andUp/src/org/openj9/test/java/security/Test_AccessControlContext.java

```diff
@@ -1,7 +1,7 @@
 package org.openj9.test.java.security;
 
 /*******************************************************************************
- * Copyright (c) 1998, 2017 IBM Corp. and others
+ * Copyright (c) 1998, 2018 IBM Corp. and others
```

Comment:
Did we modify this file? Why we need to update the copyright?

Reply:
That was a mistake.  Change reverted.